CHANGELOG.md

```diff
@@ -3,6 +3,11 @@
 
   ## v8.10.0
 
+  * **Performance: Rework timing range overlap calculations for multiple transaction segments
```

Comment:
The formatting here is missing the end **
```suggestion
  * **Performance: Rework timing range overlap calculations for multiple transaction segments**
```

Reply:
addressed with [e546d82](https://github.com/newrelic/newrelic-ruby-agent/pull/1274/commits/e546d82fe271429682a131d477f9a820e5dc60e5), thanks!